web/src/beta/lib/reearth-ui/components/Panel/index.tsx

```diff
@@ -10,26 +10,38 @@ export type PanelProps = {
   children: ReactNode;
   actions?: ReactNode;
   onCancel?: () => void;
+  dataTestid?: string;
 };
 
 export const Panel: FC<PanelProps> = ({
   title,
   width,
   children,
   actions,
-  onCancel
+  onCancel,
+  dataTestid
 }) => {
+  const uniqueId = useId();
+  const titleId = `${uniqueId}-title`;
```

Comment:
please remove anything related with id because it is generate dynamic which can break tests if not stable